app/src/main/java/ru/yandex/practicum/contacts/presentation/main/ContactUi.java

```diff
@@ -17,15 +17,14 @@ public class ContactUi implements ListDiffInterface<ContactUi> {
     private ContactUi t;
     @Override
     public boolean theSameAs(@NonNull ContactUi t){
-        return this.t.hashCode() == t.hashCode();
+        return this.hashCode() == t.hashCode();
     }
 
-/*    @Override
     public boolean equals(@NonNull ContactUi t) {
-        return ListDiffInterface.super.equals(t);
+        return this.equals(t);
```

Comment:
Тут видимо я ошибся в предыдущем комментарии. В общем такое переопределение метода не стоит делать, так как тут будет рекурсия)
Лучше переопределить вот так, вызвав метод базового класса, либо удалить в целом переопределение метода тут
```suggestion
        return super.equals(t);
```